docs/install/tutorials/rpi.rst

```diff
@@ -127,22 +127,12 @@ Use the arrow keys to navigate to the end of the file, then copy and paste the f
       static ip_address=192.168.4.1/24
       nohook resolv.conf, wpa_supplicant
 
-Then, make your current local DNS available
-
 After installing the new ``hostapd`` and ``dnsmasq`` packages and setting a static IP, you should reboot the system.
 
 .. code-block:: console
 
   sudo reboot
 
-After rebooting, you can ensure that your system is running with the static IP address by running the command ``ipconfig`` and reviewing that ``wlan0`` has the new IP address printed. It should contain this output (notice the IP address):
-
-.. code-block:: text
-
-  wlan0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
-          inet 192.168.4.1  netmask 255.255.255.0  broadcast 192.168.4.255
-          inet6 fe80::e02e:4991:29ac:f076  prefixlen 64  scopeid 0x20<link>
-
```

Comment:
I didn't get the IP to show up here even after a reboot, until I'd done all the dnsmasq/hostapd configuration, so I removed it here to avoid confusion.